.azuredevops/platformPipelines/platform.dependencies.yml

```diff
@@ -52,18 +52,18 @@ stages:
               displayName: Locks Resource Group
 
   - ${{ if eq( parameters.deployVhdDependencies, true) }}:
-    - stage: deploy_vhd
-      displayName: Store VHD to Storage Account
-      dependsOn:
-        - deploy_rg
-      variables:
-        templateFilePath: $(dependencyPath)/constructs/StoreVhdToStorage/deploy.bicep
-      jobs:
-        - template: /.azuredevops/pipelineTemplates/jobs.dependenciesModuleDeployment.yml
-          parameters:
-            deploymentBlocks:
-              - templateFilePath: $(templateFilePath)
-                displayName: Deploy module
+      - stage: deploy_vhd
```

Comment:
👻 👻 👻 
I'd suggest either to disable the YAML extension or to look for the right setting to disable to keep a cleaner commit history. Maybe something like yaml.editor.format = "false" or similar.

Reply:
disabled it

Reply:
The vhd block has one indent added. Is this intended? If not I'd suggest to rollback this change

Reply:
It's now aligned with the other `if` blocks. So I think it's actually a good automatic change :D

Reply:
Or all the others were indented by the same extension in the past 🔧

Reply:
Even if 😏😄

Reply:
🙄 😮‍💨